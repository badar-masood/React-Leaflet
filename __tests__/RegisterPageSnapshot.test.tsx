import renderWithProviders from "../jest.setup";
import RegisterPage from "../src/pages/login";

it("renders login unchanged", () => {
  const { container } = renderWithProviders(<RegisterPage />);
  expect(container).toMatchSnapshot();
});
