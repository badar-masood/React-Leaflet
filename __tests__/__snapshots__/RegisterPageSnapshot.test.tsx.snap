// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders login unchanged 1`] = `
<div>
  <div
    class="chakra-container css-1fffh61"
  >
    <h1
      class="chakra-heading css-my13z0"
    >
      Login
    </h1>
    <div
      class="chakra-stack css-owjkmg"
    >
      <img
        alt="Backpack"
        class="chakra-image css-14a85xb"
        src="/img.jpg"
      />
      <div
        class="css-njifjv"
      >
        <form
          autocomplete="off"
          novalidate=""
        >
          <div
            class="chakra-form-control css-1itv5e3"
            role="group"
          >
            <label
              class="chakra-form__label css-uvho5g"
              for="field-1"
              id="field-1-label"
            >
              username
              <span
                aria-hidden="true"
                class="chakra-form__required-indicator css-0"
                role="presentation"
              >
                *
              </span>
            </label>
            <input
              aria-required="true"
              class="chakra-input css-0"
              id="username"
              name="username"
              placeholder="your username"
              required=""
              type="text"
            />
          </div>
          <div
            class="chakra-form-control css-1itv5e3"
            role="group"
          >
            <label
              class="chakra-form__label css-uvho5g"
              for="field-2"
              id="field-2-label"
            >
              password
              <span
                aria-hidden="true"
                class="chakra-form__required-indicator css-0"
                role="presentation"
              >
                *
              </span>
            </label>
            <input
              aria-required="true"
              class="chakra-input css-0"
              id="password"
              name="password"
              placeholder="your password"
              required=""
              type="password"
            />
          </div>
          <div
            class="css-gmuwbf"
          >
            <button
              class="chakra-button css-ix34nz"
              type="submit"
            >
              Login
            </button>
          </div>
        </form>
      </div>
    </div>
    <div
      class="chakra-stack css-1xr1bob"
    >
      <p
        class="chakra-text css-0"
      >
        Don't have an account?
      </p>
      <button
        class="chakra-button css-5c2176"
        href="/register"
        type="button"
      >
        Click here to register now!
      </button>
    </div>
  </div>
</div>
`;
